astro/src/content/docs/extend/events-and-webhooks/events/user-create-complete.mdx

```diff
@@ -35,12 +35,16 @@ export const eventType = 'user.create.complete';
     The unique Id of the event. You may receive an event more than once based upon your transaction settings. This Id may be used to identify a duplicate event.
   </APIField>
 
+  <APIField slot="trailing-fields" name="event.tenantId" type="UUID">
```

Comment:
is this a duplicate entry?

Reply:
You are correct. I must have copy pasted this. Will remove.